packages/rum/src/domain/record/serialization/serializeNode.spec.ts

```diff
@@ -586,11 +557,17 @@ describe('serializeNodeWithId', () => {
     })
 
     describe('<link rel="stylesheet"> elements', () => {
+      let originalStyleSheets: StyleSheetList
+
+      beforeEach(() => {
+        originalStyleSheets = document.styleSheets
+      })
+      afterEach(() => {
+        Object.defineProperty(document, 'styleSheets', { value: originalStyleSheets, configurable: true })
+      })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏the `styleSheets` property is defined on a `document` prototype, that's why you failed to restore it with `const originalDescriptor = Object.getOwnPropertyDescriptor(document, 'styleSheets')`.

You could simply `delete document.styleSheets` to remove the mocked property defined in tests.